pkgs/top-level/all-packages.nix

```diff
@@ -23780,4 +23780,6 @@ in
     stdenv = crossLibcStdenv;
   };
 
+  pforth = callPackage ../development/compilers/pforth {};
```

Comment:
Sort this into a better category in this file, see notes at the top, should be somewhat alphabetical

Reply:
Done